app/views/pages/i-belong.html.erb

```diff
@@ -36,25 +60,24 @@
             </ul>
           </div>
           <div class="lp-support__card white-bg lp-support__card--bordered">
-            <% if current_user %>
-              <% if Programme.i_belong_certificate.user_enrolled?(current_user) %>
-                <h2 class="govuk-heading-m">Completing the programme</h2>
-                <ol class="govuk-body">
-                  <li>Complete our ‘Encouraging Girls into GCSE Computer Science’ short course</li>
-                  <li>Participate in a range of recommended activity</li>
-                  <li>Claim your school certificate</li>
-                </ol>
-                <%= link_to 'Visit dashboard', dashboard_path, class: 'button govuk-button govuk-!-margin-bottom-0' %>
-              <% else %>
-                <h2 class="govuk-heading-m">Where to start?</h2>
-                <ol class="govuk-body">
-                  <li>Enrol on the programme</li>
-                  <li>Obtain your free handbook and posters</li>
-                  <li>Complete our ‘Encouraging Girls into GCSE Computer Science’ short course</li>
-                  <li>Participate in a range of recommended activity</li>
-                </ol>
-                <%= link_to 'Enrol on the programme', programme.enrol_path(user_programme_enrolment: { user_id: current_user.id, programme_id: programme.id }), class: 'button govuk-button govuk-!-margin-bottom-0' %>
-              <% end %>
+            <% case state %>
+            <% when :enrolled  %>
```

Comment:
I'm wondering if in your above when statement, if you could have a variable for `title` `steps` `link_cta` and `link_url` as the template looks identical and then plonk them into just one set of html?